parsers/raku.c

```diff
@@ -321,21 +322,20 @@ findPerl6Tags (void)
         }
     }
 
-    deinitP6Ctx(&ctx);
+    deinitRakuCtx(&ctx);
 }
 
 parserDefinition *
-Perl6Parser (void)
+RakuParser (void)
 {
-    static const char *const extensions[] = { "p6", "pm6", "pm", "pl6",
-		"t6", "raku", "rakumod", "rakutest", NULL };
+    static const char *const extensions[] = { "raku", "rakumod", "rakutest", "rakudoc", NULL };
```

Comment:
ctags should support older code.
With this change, ctags cannot handle Perl6 files having '.pl6' as a suffix.
This is degrading.